### PR TITLE
Show less no longer splits words.  Add to tour button changes to Added!

### PR DESCRIPTION
**Pull request by jamiander**

### CODE REVIEW

File: src/main/resources/static/script.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ function showMoreText() {
 
         if(content.length > showChar) {
             var c = content.substr(0, showChar);
-            var h = content.substr(showChar, content.length - showChar);
-            var html = c + '<span class="moreellipses">' + ellipsestext+ '&nbsp;</span><span class="morecontent"><span>' + h + '</span>&nbsp;&nbsp;<a href="" class="morelink">' + moretext + '</a></span>';
+            var end = Math.min(c.length, c.lastIndexOf(" "));
```
Comment (KBeas21) [inline src/main/resources/static/script.js:12]: What is this Math.min doing? Is it going to either split the word at the end of itself or the space (" ") after it? I'm not 100% sure that this is needed.

File: src/main/resources/templates/gardens/list.html [hunk 2]
```diff
@@ -82,7 +82,8 @@ <h5 class="card-title" th:text="${garden.name}" ></h5>
                                 <a class="card-text btn-sm btn-primary" th:href="@{'/user/view/' + ${garden.id}}">More Information</a>
                                 <br>
                                 <div style="margin-top:5px">
-                                    <a class="card-text btn-sm btn-success" id="add" th:href="@{'/user/gardens/tour/' + ${garden.id}}" style="margin-right:10px;">Add to Tour</a>
+                                    <a class="card-text btn-sm btn-success" id="added" style="margin-right:10px; disabled;" th:if="${tourGardens.contains(garden)}">Added!</a>
+                                    <a class="card-text btn-sm btn-success" id="add" th:href="@{'/user/gardens/tour/' + ${garden.id}}" style="margin-right:10px;" th:unless="${tourGardens.contains(garden)}">Add To Tour</a>
```
Comment (KBeas21) [inline src/main/resources/templates/gardens/list.html]: Glad this worked shout out to Ryan for the help!

File: src/main/resources/templates/gardens/list.html [hunk 3]
```diff
@@ -82,7 +82,8 @@ <h5 class="card-title" th:text="${garden.name}" ></h5>
                                 <a class="card-text btn-sm btn-primary" th:href="@{'/user/view/' + ${garden.id}}">More Information</a>
```
Comment (hstrodtman) [inline src/main/resources/templates/gardens/list.html:82]: Not sure if its the JS or the button changes, but the Add to Tour and View on Map buttons were next to each other and now they are above/below with no spacing before the garden is in a tour.

File: src/main/resources/static/script.js (hunk 1 above)
Comment (jamiander) [reply src/main/resources/static/script.js:12]: My understanding is that it takes the shorter of the two options, either the 75 characters or the spot closest to 75 characters where " " is the last index.  Does that make sense?  There's probably a better way to do this?

File: src/main/resources/templates/gardens/list.html (hunk 3 above)
Comment (jamiander) [reply src/main/resources/templates/gardens/list.html:82]: Hmmm...mine are side by side.  I had to fix another bug from the add a garden controller, so I'll push again.  Maybe I changed something?  I'll check it out more.

**Review by KBeas21 — APPROVED** [review]

**Review by hstrodtman — APPROVED** [review]

**Pull request merged**